karaf/pom.xml

```diff
@@ -48,6 +48,16 @@
     <servicemix.version>6.0.0</servicemix.version>
     <reflections.bundle.version>0.9.9_1</reflections.bundle.version> <!-- see reflections.version -->
 
+    <!-- brooklyn-rest-client's transitive dependencies from resteasy -->
+    <jboss-annotations-api_1.1_spec.version>1.0.1.Final</jboss-annotations-api_1.1_spec.version>
+    <javax-activation.version>1.1</javax-activation.version>
+    <jcip-annotations.version>1.0</jcip-annotations.version>
+    
+    <!-- brooklyn-rest-client's transitive dependencies from httpclient:4.4.1 -->
+    <commons-logging.version>1.2</commons-logging.version>
+    <commons-codec.version>1.9</commons-codec.version>
+    <commons-io.version>2.4</commons-io.version>
+
```

Comment:
I think we are still adding those to the root pom for consistency. The versions here are used in the build process/tests only.